README.md

```diff
@@ -44,9 +44,10 @@ Also, make sure all hostnames are correct for the environment you're running in.
 
 ```shell
 source .env
-NEO4J_HOST=localhost
-ELASTIC_API_HOST=localhost
-REDIS_HOST=localhost
+export $(cut -d= -f1 .env)
```

Comment:
When standing up the environment for the first time locally, I noticed that while the environment variables existed in the shell i was in, they weren't listed when using `env` and it also seemed the `dug` seed/crawl command wasn't picking them up. I came across this while debugging: `Environment variables set without export will not be inherited in the environment of the commands you are calling.`[1]. Open to ideas and/or if there's a better approach.


[1] https://askubuntu.com/questions/205688/whats-the-difference-between-set-export-and-env-and-when-should-i-use-each